components/advocates/AdvocatesJoinSection.vue

```diff
@@ -1,28 +1,23 @@
 <template>
   <section class="join-section">
-    <article
-      class="
-        join-section__feature
-        join-section__intro
-      "
-    >
-      <p
-        class="
-          copy__paragraph
-          join-section__intro-copy
-        "
+    <div class="bx--row">
+      <article
+        class="bx--col-lg-4 bx--col-md-4 join-section__feature join-section__intro"
```

Comment:
As discussed offline, since `bx--col-padding` doesn't appear to do exactly the same (it add a little more space to the edges of the elements as well), we decided to leave the implementation as is and address it in the future with a more general refactor of recurrent classes (#1677).